afl.pyx

```diff
@@ -75,10 +75,13 @@ def _hash(key, offset):
 
 cdef object trace
 def trace(frame, event, arg):
-    global prev_location
+    global prev_location, tstl_mode
     cdef unsigned int location, offset
+    cdef object filename = frame.f_code.co_filename
```

Comment:
(Declaring a local variable as `cdef object` is not necessary, but it doesn't hurt either.)